alacritty_terminal/src/ansi.rs

```diff
@@ -1119,29 +1121,25 @@ where
             }};
         }
 
-        match byte {
-            b'B' => configure_charset!(StandardCharset::Ascii),
-            b'D' => self.handler.linefeed(),
-            b'E' => {
+        match (byte, intermediates.get(0)) {
+            (b'B', num) => configure_charset!(StandardCharset::Ascii, num),
```

Comment:
We shouldn't name the intermediate `num`, since we're usually dealing with characters and that just gets confusing.

Just naming it `intermediate` should be good, even if that means we'll have to linebreak.